packages/flutter/lib/src/rendering/list_wheel_viewport.dart

```diff
@@ -759,27 +762,27 @@ class RenderListWheelViewport
       _layoutChild(lastChild!, childConstraints, ++currentLastIndex);
     }
 
-    offset.applyViewportDimension(_viewportExtent);
-
     // Applying content dimensions bases on how the childManager builds widgets:
     // if it is available to provide a child just out of target range, then
     // we don't know whether there's a limit yet, and set the dimension to the
     // estimated value. Otherwise, we set the dimension limited to our target
     // range.
+    // Test by 'infinite scrolling child builder with lower and upper limits' at
+    // 'list_wheel_scroll_view_test.dart'.
     final double minScrollExtent = childManager.childExistsAt(targetFirstIndex - 1)
       ? _minEstimatedScrollExtent
       : indexToScrollOffset(targetFirstIndex);
     final double maxScrollExtent = childManager.childExistsAt(targetLastIndex + 1)
-        ? _maxEstimatedScrollExtent
-        : indexToScrollOffset(targetLastIndex);
+      ? _maxEstimatedScrollExtent
+      : indexToScrollOffset(targetLastIndex);
     offset.applyContentDimensions(minScrollExtent, maxScrollExtent);
```

Comment:
Adjust the content dimensions for there is a special[ test case](https://github.com/doxuanviet1996/flutter/blob/89452a972a5ba3cdb6720a438f43ff5d6082fa69/packages/flutter/test/widgets/list_wheel_scroll_view_test.dart#L159)

Reply:
The logic here is difficult to understand, so I linked this test case in the comments.